aws/resource_aws_iam_group_policy_test.go

```diff
@@ -139,6 +139,28 @@ func TestAccAWSIAMGroupPolicy_generatedName(t *testing.T) {
 	})
 }
 
+func TestAccAWSIAMGroupPolicy_importBasic(t *testing.T) {
```

Comment:
Rather than creating a separate import test, which duplicates test code/infrastructure and does not test all resource configurations, we prefer to now add the import `TestStep` to all applicable existing tests. See also: https://github.com/terraform-providers/terraform-provider-aws/issues/8944

Here this means removing the new `TestAccAWSIAMGroupPolicy_importBasic` and `testAccAwsIamGroupPolicyConfig` while copying the code block noted below to each existing acceptance test:

```go
			{
				ResourceName:      resourceName,
				ImportState:       true,
				ImportStateVerify: true,
			},
```

Reply:
Done. Just skipped the `_disappears`.